### DISCUSSION TITLE
Path to disabling MESS

### DISCUSSION THREAD

**Discussion by stevanlohja** [ethereumclassic/ECIPs#492] [ECIPs]
Will there be a path to disabling MESS?

**Comment by ghost** [comment]
That is something I believe needs to happen eventually, for now I would like to see miner migration after ETH merges to 2.0 but it is fair to start this discussion.

**Comment by bobsummerwill** [comment]
I would like to see MESS disabled too, but there is no specific plan just yet.

A big influx of hash post-Merge would make that less scary.

**Comment by OmniEdge** [comment]
Once ETH finally moves to POS more GPU miners will migrate to ETC and make it safer to disable MESS.

MESS was designed as an emergency. ETC is following the nakamoto consensus and needs to continue this and the issue is that MESS  delays the nakamoto consensus at the network level.

**Comment by TheCrowbill** [comment]
If MESS was implemented solely as an emergency countermeasure to the 51% attacks, it stands to reason that it should be removed when the emergency is over. At what point will we declare that the emergency has ended? The expected post-Merge hash influx will not happen as a single point event but rather could take months (years?) for hashrates to stabilize. Can a target date/block for MESS deactivation even be considered yet considering such uncertainty?

**Reply by bobsummerwill** [reply]
I think we just need to evaluate the situation after The Merge.

I cannot see that being a particularly slow migration - certainly not months or years.  Miners need to make decisions at or after The Merge, unless they can afford to sit and wait with no income.

So maybe a month after The Merge we reassess?

**Comment by diega** [comment]
Please follow up at [discussion#522](https://github.com/orgs/ethereumclassic/discussions/522)
